docs/guides/vpn/dual-VPN.md

```diff
@@ -32,21 +32,27 @@ Make sure that the DNS requests go though the instance of OpenVPN:
 push "dhcp-option DNS 10.9.0.1"
 ```
 
-One other setting that we need to change is to comment out `# push "redirect-gateway def1 bypass-dhcp"`. Commenting out this line, ensures that no traffic is routed via the VPN server.
+One other setting that we need to change is to comment out the `bypass-dhcp` instruction so that it looks like below: 
+
+```
+# push "redirect-gateway def1 bypass-dhcp"`.
+``` 
+
+Commenting out this line, ensures that no traffic is routed via the VPN server.
 
 Save the file and start the second instance of OpenVPN:
 
-```
+```bash
 systemctl start openvpn@server2.service
 ```
 
-*If your distribution does not have `systemctl` you may use commands like below to start OpenVPN with your second configuration as a daemon:* 
+*If your distribution does not have `systemctl` you may use commands below to start OpenVPN with your second configuration as a daemon:* 
```

Comment:
There is only one command following this line, so it should not say there are multiple commands.